RibonDAO/app
# Feat/prod3 352

# Description

- Adds cases for when user is already logged, but not authenticated

## Type of change


## How to test

- [if you're signed in, log out, then] **login via magic link on the menu**, then
- try to **donate**
- check that you're redirected to a **sign in page** that talks about an extra ticket
- insert a **different email** than the one you used to log in via magic link and try to login via Apple or Google
- ensure that it shows an **error modal** in case the email does not match
- put the **correct email**
- check that:
   1. google/apple: after you sign in, you get **redirected** to causes page and get an extra ticket **email**
   2. magic link: after you choose magic link, you get **redirected** to a page that tells you you will get an email; you get the **email**
